action.yml

```diff
@@ -63,4 +63,4 @@ runs:
         $body = (ConvertTo-Json $annotation -Compress) -replace '(\\+)"', '$1$1"' -replace "`"", "`"`""
         Write-Host "Request Body:"
         Write-Host $body
-        az rest --method put --uri "$($aiResourceId)/Annotations?api-version=2015-05-01" --body "$($body)" --headers "{'Content-Type':'application/json'}"
+        az rest --method put --uri "$($aiResourceId)/Annotations?api-version=2015-05-01" --body "$($body)" --headers "Content-Type=application/json"
```

Comment:
Oh.. haha! I should have caught that before I approved this!

Since the line had the word "json" in it, I guess my brain switched to JSON syntax.